Microsoft.Toolkit.Uwp.UI.Controls/AdvancedCollectionViewSource/VectorChangedEventArgs.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Windows.Foundation.Collections;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.AdvancedCollectionViewSource
+{
+    /// <summary>
+    /// Vector changed EventArgs
+    /// </summary>
+    public class VectorChangedEventArgs : IVectorChangedEventArgs
```

Comment:
I think this class can be kept internal. It's enough to have IVectorChangedEventArgs public which is already part of WinRT